src/misc_tools.c

```diff
@@ -538,15 +538,15 @@ size_t get_group_self_nick_truncate(Tox *tox, char *buf, uint32_t groupnum)
     return len;
 }
 
-/* copies data to msg buffer, removing return characters.
-   returns length of msg, which will be no larger than size-1 */
 size_t copy_tox_str(char *msg, size_t size, const char *data, size_t length)
 {
     size_t j = 0;
 
     for (size_t i = 0; (i < length) && (j < size - 1); ++i) {
-        if (data[i] != '\r') {
-            msg[j] = data[i];
+        const char ch = data[i];
+
+        if (ch != '\r' && ch != '\t' && ch != '\v') {
```

Comment:
I guess another option would be to replace them with space characters. Might make more sense in some scenarios?

Reply:
Makes sense for \t and \v, but not really for \r since that's usually at the end of a line.